cli/src/commands/squash.rs

```diff
@@ -172,6 +173,13 @@ pub(crate) struct SquashArgs {
     #[arg(long)]
     editor: bool,
 
+    /// Do not open an editor to combine descriptions
+    ///
+    /// Useful for non-interactive runs (agents, scripts). Accepts the default combined description
+    /// instead of launching an editor when multiple commit messages need to be merged.
+    #[arg(long, conflicts_with = "editor")]
+    no_editor: bool,
```

Comment:
Isn't it better to rename this to `use-combined-message`? We already have `--use-destination-message` to disable editor interaction, and I think it's unusual that combined message can be used without manual editing.